src/general/OptionsMenu.tscn

```diff
@@ -1916,11 +1919,51 @@ selected = 0
 margin_top = 667.0
 margin_right = 702.0
 margin_bottom = 671.0
+rect_pivot_offset = Vector2( -571, -253 )
```

Comment:
Why is this property set?

Reply:
Godot being godot again. It really likes changing random things. I could revert, but that does likely do nothing.

Reply:
If anyone ever applies any rotation or maybe something else can trigger this, someone is going to have a really bad time trying to figure out why that doesn't work on this one node.